### PR TITLE
[doc] Replace image on the private envinronment page in GS and fix text

### PR DESCRIPTION
**Pull request by Zhbert**
## Description

* The image of the deployment scheme in a private environment in Getting Started has been changed: added an internal repository of packages for the OS.
* Added the Russian version of the scheme

## Why do we need it, and what problem does it solve?

This will help solve problems with installing curl in a private environment: the requirement of an internal repository is now immediately visible to the user.

## Why do we need it in the patch release (if we do)?

To reduce possible confusion in for a reader.

## What is the expected result?

The Getting Started is consistent and clean.

## Checklist
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: chore
summary: Make the Getting Started more consistent.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA.liquid [hunk 1]
```diff
@@ -44,6 +44,8 @@ Installation in a private environment is generally similar to installation on [b
 
 Scheme of Deckhouse installation in a private environment:<br />
 <img src="/images/gs/private-env-schema.svg" alt="Scheme of Deckhouse installation in a private environment">
+
+> The internal repository of OS packages is necessary to install curl on the nodes of the future cluster (if there is no possibility of accessing the official repositories through a proxy server).
```
Comment (kirkonru) [inline docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA.liquid:48]:
```suggestion
> An internal repository of OS packages is necessary to install curl on future cluster nodes (if there is no access to the official repositories via a proxy server).
```

File: docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA_RU.liquid [hunk 2]
```diff
@@ -44,6 +44,8 @@
 
 Схема развертывания Deckhouse в закрытом окружении:<br />
 <img src="/images/gs/private-env-schema.svg" alt="Схема развертывания Deckhouse в закрытом окружении">
+
+> Внутренний репозиторий пакетов ОС необходим для установки curl на узлах будущего кластера (при отсутствии возможности доступа к официальным репозиториям через прокси-сервер).
```
Comment (z9r5) [inline docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA_RU.liquid:48]:
```suggestion
{% alert %}Внутренний репозиторий пакетов ОС необходим для установки curl на узлах будущего кластера (при 
отсутствии возможности доступа к официальным репозиториям через прокси-сервер).
{% endalert %}
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Zhbert. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Review by z9r5 — APPROVED** [review]

**Pull request merged**